taskfiles/lint/python.yaml

```
@@ -0,0 +1,66 @@
+version: "3"
+
+tasks:
+  check-py:
+    cmds:
+      - task: "check-py-format"
+      - task: "check-py-static"
+
+  fix-py:
+    cmds:
+      - task: "fix-py-format"
+      - task: "fix-py-static"
+
+  check-py-format:
+    cmds:
+      - task: "py"
+        vars:
+          LINT_TOOL: "ruff"
+          COMMAND: "format"
+          FLAGS: "--diff"
+
+  fix-py-format:
+    cmds:
+      - task: "py"
+        vars:
+          LINT_TOOL: "ruff"
+          COMMAND: "format"
+
+  check-py-static:
+    cmds:
+      - task: "py"
+        vars:
+          LINT_TOOL: "ruff"
+          COMMAND: "check"
+      - task: "py"
+        vars:
+          LINT_TOOL: "mypy"
+
+  fix-py-static:
+    cmds:
+      - task: "py"
+        vars:
+          LINT_TOOL: "ruff"
+          COMMAND: "check"
+          FLAGS: "--fix"
+      - task: "py"
+        vars:
+          LINT_TOOL: "mypy"
+
+  py:
+    internal: true
+    vars:
+      COMMAND: >-
+        {{default "" .COMMAND}}
+      FLAGS: >-
+        {{default "" .FLAGS}}
+      PY_LINT_DIRS:
+        - "src/log_archival_bench"
+        - "tests"
+    requires:
+      vars: ["LINT_TOOL"]
+    dir: "{{.ROOT_DIR}}"
+    cmds:
+      - for:
+          var: "PY_LINT_DIRS"
+        cmd: "uv run {{.LINT_TOOL}} {{.COMMAND}} {{.FLAGS}} {{.ITEM}}"
```

How about merging `lint_tool` `command` `flags` into `command`?

```suggestion
  fix-py-static:
    cmds:
      - task: "py"
        vars:
          COMMAND: "ruff check --fix"
      - task: "py"
        vars:
          COMMAND: "mypy"

  py:
    internal: true
    vars:
      PY_LINT_DIRS:
        - "src/log_archival_bench"
        - "tests"
    requires:
      vars: ["COMMAND"]
    dir: "{{.ROOT_DIR}}"
    cmds:
      - for:
          var: "PY_LINT_DIRS"
        cmd: "uv run {{.COMMAND}} {{.ITEM}}"
```

---

Deferring to @kirkrodrigues 's review

---

I'm open to either approach. The single `.COMMAND` variable is less structured since it uses a raw string for commands. However, the original approach's `.FLAGS` could also introduce spaces in a string.